pyproject.toml

```diff
@@ -14,6 +14,199 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+[build-system]
+requires = ["scikit-build-core>=0.7.0"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "tvm"
+version = "0.16.0.dev0"
```

Comment:
The latest version might be 0.22.0.dev0 according to https://github.com/apache/tvm/tags